test/plugins/webhooks.helper.test.js

```diff
@@ -1966,35 +1997,68 @@ describe('startHookEvent test', () => {
                         search: { field: 'scmUri', keyword: 'github.com:789123:%' },
                         params: { state: 'ACTIVE' }
                     })
-                    .resolves([]);
+                    .resolves([
+                        getPipelineMocks({
+                            id: 'pipelineHash',
+                            scmUri: 'github.com:789123:master',
+                            annotations: {},
+                            admins: {
+                                baxterthehacker: false
+                            },
+                            workflowGraph,
+                            branch: Promise.resolve('master')
+                        })
+                    ]);
+                const pipelineMock2 = _.cloneDeep(pipelineMock);
```

Comment:
you can write a simple deep clone instead of adding the entire lodash pkg I think